### PR TITLE
Add new ebook - Why Your CMS Project Will Fail

### PR DESCRIPTION
**Pull request by jazzdrive3**
Updated with images. Ready for final review.

### CODE REVIEW

File: src/ebooks/why-your-cms-will-fail/chapters/chapter-2.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'One: A lack of stakeholder involvement'
```
Comment (sirkitree) [inline src/ebooks/why-your-cms-will-fail/chapters/chapter-2.md:2]: Chapters are currently numbered automatically, so this sort of looks funny on the table of contents:
https://pr74-dex8gkuvg6cggqol1lwasfiqtddsrwez.tugboatqa.com/ebooks/why-your-cms-will-fail/table-of-contents/
image

Comment (sirkitree) [reply src/ebooks/why-your-cms-will-fail/chapters/chapter-2.md:2]: Kinda reads funny on the actual pages as well
image

File: src/ebooks/why-your-cms-will-fail/chapters/chapter-7.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'Five: Not knowing how to use the system'
```
[the remark this answers is not in the document]
Comment (sirkitree) [reply src/ebooks/why-your-cms-will-fail/chapters/chapter-7.md:2]: image

File: src/ebooks/why-your-cms-will-fail/index.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Sample eBook"
```
Comment (sirkitree) [inline src/ebooks/why-your-cms-will-fail/index.html:2]: I suggest putting the actual title here as well.

**Review by sirkitree — CHANGES_REQUESTED** [review]
I know you're still working on it, but thought I'd give some feedback early.

File: src/ebooks/why-your-cms-will-fail/chapters/chapter-2.md (hunk 1 above)
Comment (jazzdrive3) [reply src/ebooks/why-your-cms-will-fail/chapters/chapter-2.md:2]: This is related to an issue I opened about how this currently works.

https://github.com/Lullabot/lb-ebook-poc/issues/72

**Review by sirkitree — APPROVED** [review]
i think this looks good. I have some pretty sweeping changes coming up in the branch I'm working on locally to address #72 but I think if you merge this I can then make those changes (mostly naming), and you can indicate which of these should be chapters in that issue.

If you want to indicate which ones are chapters in this PR, you can update the ones you want with front-matter, adding `tags: [chapter]`

**Comment by jazzdrive3** [comment]
This should be fine as-is, with no changes. Everything works as a chapter for this one.

**Pull request merged**